app/components/landing-page/template.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="reviews-header">
+    <div class="container">
+        <div class="row">
+            <div class="text-center m-t-lg col-xs-12">
+                <div class="m-b-lg reviews-brand"></div>
+                {{#if showSetup}}
+                    {{start-moderating action=(action setupProvider) providers=providersToSetup}}
```

Comment:
Same thing here: It seems cleaner for start-moderating to use the router service and trigger the transition itself, rather than passing `setupProvider` all the way down from the controller.